### PR TITLE
Add python-crypt4gh recipe

### PR DESCRIPTION
**Pull request by jmfernandez**
This pull requests adds a new recipe for python crypt4gh 1.5 package (https://github.com/EGA-archive/crypt4gh).

The package is one of the reference implementations of [GA4GH encryption standard](https://www.ga4gh.org/news/crypt4gh-a-secure-method-for-sharing-human-genetic-data/), which is adopted by EGA.

The package is currently being maintained by both @silverdaz and @blankdots , I'm a mere user of it.

### CODE REVIEW

**Comment by jmfernandez** [comment]
@bioconda/core , could you review and merge this PR ,  please?

File: recipes/python-crypt4gh/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    - crypt4gh-keygen --sk bob.key --pk bob.pub -C 'bioconda Bob test' --nocrypt
+    - crypt4gh encrypt --sk bob.key --recipient_pk bob.pub --recipient_pk alice.pub < testfile-help.txt > message.c4gh
+    - crypt4gh decrypt --sk alice.key < message.c4gh > message.alice.received
+    - diff testfile-help.txt message.alice.received
+    - crypt4gh decrypt --sk bob.key < message.c4gh > message.bob.received
+    - diff testfile-help.txt message.bob.received
+
+about:
+  home: https://github.com/EGA-archive/crypt4gh
+  license: Apache Software License
```
Comment (cbrueffer) [inline recipes/python-crypt4gh/meta.yaml:48]:
```suggestion
  license: Apache-2.0
```
valid SPDX identifier

**Review by cbrueffer — COMMENTED** [review]
Looks good, just a small license suggestion.

**Review by cbrueffer — APPROVED** [review]

**Comment by cbrueffer** [comment]
@bioconda-bot please merge

**Pull request merged**